model/infinispan/src/test/java/org/keycloak/models/sessions/infinispan/initializer/ConcurrencyVersioningTest.java

```diff
@@ -234,19 +234,16 @@ public static void endBatch(Cache<String, String> cache) {
 
 
     protected DefaultCacheManager getVersionedCacheManager() {
-        GlobalConfigurationBuilder gcb = new GlobalConfigurationBuilder();
-
-
-        boolean clustered = false;
         boolean async = false;
-        boolean allowDuplicateJMXDomains = true;
+        boolean clustered = false;
```

Comment:
I'm wondering what is purpose of these boolean variables across *Test.java classes when they are never changed. They could be probably eliminated and code could be simplified.

Reply:
Right, great catch, thanks Martin! Previously tried to do the minimal necessary amount of changes to get the code to compile with new Wildfly / Wildfly Core version. But you are correct regarding those two variables (assuming they got there as the code for new test got copied from `DefaultInfinispanConnectionProviderFactory.java` [in this file those two can be initialized from system properties] and assuming as the code of these tests got simplified / reduced during the time, their use doesn't make much sense already). 

In any case, dropped their use if / where applicable (in tests touched within this PR). Please see the latest version.